src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/MemberExpansion.cs

```diff
@@ -132,6 +148,11 @@ internal sealed class MemberExpansion : Expansion
                 }
             }
 
+            if (dynamicViewExpansion != null)
+            {
+                expansions.Add(dynamicViewExpansion);
+            }
```

Comment:
Yes, we expect to show all "regular" expansions and just append Dynamic View (unless the ", dynamic" format specifier is used).